validator/src/main.rs

```diff
@@ -950,6 +954,32 @@ pub fn main() {
                 /* .default_value() intentionally not used here! */
                 .help("Keep this amount of shreds in root slots."),
         )
+        .arg(
+            Arg::with_name("shred_storage")
```

Comment:
`—shred-storage` seems to not be quite what this option is about.  Perhaps `—rocksdb-shred-compaction [level|fifo]`?

Reply:
Using `shred-storage` is to reserve the possibility that we have completely different storage for shreds in the future, but yep for now it's only switching between RocksDB's compaction styles.

Reply:
Thanks, and yeah there's no need to prematurely generalize arguments.  Who knows what the future holds.